cmds/ocm/commands/ocicmds/artifacts/transfer/cmd.go

```diff
@@ -152,7 +152,10 @@ func NewAction(ctx clictx.Context, session oci.Session, target string, transferR
 }
 
 func (a *action) Add(e interface{}) error {
-	src := e.(*artifacthdlr.Object)
+	src, ok := e.(*artifacthdlr.Object)
+	if !ok {
+		return fmt.Errorf("failed type assertion for type %T to artifacthtlr.Object", e)
```

Comment:
Basically this would be a local programming error. which is directly detected by the test cases, it is not a library function. So, I'm not sure, whether these runtime checks are really required. But anyway, the other locations should use this `%T`, also, to get a glimpse on what went wrong.